src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageRectangleSelection/ImageRectangleSelection.js

```diff
@@ -82,11 +90,11 @@ class ImageRectangleSelection extends React.Component<Props> {
         onChange(data ? this.scaledDataToNatural(data) : undefined);
     };
 
-    @computed get minDimensions() {
+    @computed get scaledMinDimensions() {
         const {minHeight, minWidth, containerHeight, containerWidth} = this.props;
 
-        let height = minHeight;
-        let width = minWidth;
+        let height = minHeight ? this.naturalVerticalToScaled(minHeight) : undefined;
```

Comment:
if the image is scaled down, we need to scale the min-dimensions too. for example, if the natural width of the image is 1000, the image format width is 500 and the image with in the overlay is 200, we still want to be able to adjust the width of the crop to 1/5 of the image.